src/store/singing.ts

```diff
@@ -936,8 +980,10 @@ export const singingStore = createPartialStore<SingingStoreTypes>({
   },
 
   SELECT_TRACK: {
+    // 選択中のトラックのノートだけを選択中として扱うため、トラックが切り替わったときは選択中のノートをクリアする。
```

Comment:
ここちょっと微妙にニュアンスが違ってて、「これだったらフィルターでも良いか」ってなっちゃうかも
たぶん、こう。

```suggestion
    // トラックを切り替えるときに選択中のノートをクリアする。
```

ちなみに理由はたぶん明確に合意が取れてないんですが、「選択トラックを戻した時にノートが選択されていない状態にするため」とかかな。。。。。むずい。。。